Src/PerformanceCollector/Shared/QuickPulseTelemetryModule.cs

```diff
@@ -128,9 +143,10 @@ public void Initialize(TelemetryConfiguration configuration)
                         QuickPulseEventSource.Log.ModuleIsBeingInitializedEvent(
                             string.Format(
```

Comment:
The whole purpose of event source is to avoid string formatting. Please move arguments to the eventsource message